src/TestFramework/AbstractTestFrameworkAdapter.php

```diff
@@ -136,6 +136,23 @@ private function makeExecutable(string $frameworkPath, bool $includeArgs = true,
             );
         }
 
+        /*
+         * Run an executable as it is if we're using a standard CLI and
+         * there's a standard interpreter available on PATH.
+         *
+         * This lets folks use, say, a bash wrapper over phpunit.
+         */
+        if ('cli' == PHP_SAPI && empty($phpExtraArgs) && is_executable($frameworkPath) && `command -v php`) {
```

Comment:
Could you explain what  `command -v php` means/does?

Reply:
Ah, i never knew using backticks like that was possible.

http://php.net/manual/en/language.operators.execution.php